dev-packages/e2e-tests/maestro/feedback.yml

```diff
@@ -3,4 +3,17 @@ jsEngine: graaljs
 ---
 - runFlow: utils/launchTestAppClear.yml
 
-- runFlow: feedback/happyFlow.yml
+
+# The following tests are happy path tests for the feedback widget on both iOS and Android.
+# They verify that the feedback form can be opened, filled out, and submitted successfully.
+# The tests are separate because iOS tests work better with `testID` and Android tests work better with `text`.
```

Comment:
The Android Test is super clean, sad that is doesn't work for iOS.

Is it flaky or the elements are not found?

Reply:
[On iOS legacy text lookups fail](https://github.com/getsentry/sentry-react-native/actions/runs/14757306115/job/41430627531?pr=4604) consistently :(